e2e/src/usecases/Assets.js

```diff
@@ -0,0 +1,174 @@
+import { generateMnemonic } from '@celo/cryptographic-utils'
+import { DEFAULT_RECIPIENT_ADDRESS, SAMPLE_BACKUP_KEY } from '../utils/consts'
+import { launchApp } from '../utils/retries'
+import { quickOnboarding, waitForElementByIdAndTap, waitForElementId } from '../utils/utils'
+
+async function validateSendFlow(tokenSymbol) {
+  // navigate to send amount screen to ensure the expected token symbol is pre-selected
+  await waitForElementId('SendSearchInput')
+  await element(by.id('SearchInput')).tap()
+  await element(by.id('SearchInput')).replaceText(DEFAULT_RECIPIENT_ADDRESS)
+  await element(by.id('SearchInput')).tapReturnKey()
+  await expect(element(by.text('0xe5f5...8846')).atIndex(0)).toBeVisible()
+  await element(by.text('0xe5f5...8846')).atIndex(0).tap()
+  await expect(
+    element(by.text(tokenSymbol).withAncestor(by.id('TokenPickerSelector')))
+  ).toBeVisible()
+  await element(by.id('BackChevron')).tap()
+  await element(by.id('Times')).tap()
+}
+
+async function validateAddFlow(tokenSymbol) {
+  // check the header includes the appropriate token symbol
+  await waitFor(element(by.id('HeaderTitle')))
+    .toHaveText(`Add ${tokenSymbol}`)
+    .withTimeout(10 * 1000)
+  await element(by.id('BackChevron')).tap()
+}
+
+const makeAssetsTests =
```

Comment:
could this be done with a describe.each instead? my IDE doesn't let me run tests individually in the UI if they are wrapped in a custom function like this

Reply:
so it makes development and debugging a bit more cumbersome